e2e/expected/nodeTreeToSketchPage/visibility.html.asketch.json

```diff
@@ -64,106 +64,6 @@
           "rotation": 0,
           "shouldBreakMaskChain": false,
           "layers": [
-            {
```

Comment:
@KimDal-hyeong why we do have changes over here. Are you sure that e2e test are passing?

Reply:
Yes, now e2e is rendering two invisible divs from PR reasons in visibility test. 
This change is correct.

<img width="544" alt="2018-10-23 11 00 28" src="https://user-images.githubusercontent.com/16788797/47330483-5a766a00-d6b3-11e8-84d1-901de1b3a0b9.png">
<img width="670" alt="2018-10-23 11 01 13" src="https://user-images.githubusercontent.com/16788797/47330484-5b0f0080-d6b3-11e8-83c2-4d5904360a10.png">